src/cryptography/hazmat/backends/openssl/aead.py

```diff
@@ -16,15 +16,15 @@
         ChaCha20Poly1305,
     )
 
-    _AEAD_TYPES = typing.Union[
+    _AEADTypes = typing.Union[
```

Comment:
This should definitely retain underscore, as it's inside an `if TYPE_CHECKING:` conditional block.